examples/00_basic/app.py

```diff
@@ -1,60 +1,121 @@
 import os
 from flask import Flask, jsonify, request, make_response
+from flask_sqlalchemy import SQLAlchemy
 from functools import wraps
+from passlib.hash import sha256_crypt as sc
+import uuid
+import secrets
 import jwt
 import datetime
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = os.environ.get('Secret')
+app.config['SQLALCHEMY _DATABASE_URI'] = 'something'
 
+db = SQLAlchemy(app)
+blacklist = []
+class User(db.Model):
+    ''' Setting up User'''
+    id = db.Column(db.Integer, primary_key=True)
+    # User vereification
+    email = db.Column(db.String(300), nullable=False, unique=True)
+    username = db.Column(db.String(60), nullable=False, unique=True)
+    password = db.Column(db.String(300), nullable=False)
+    active = db.Column(db.String(50),nullable=False)
+    token = db.Column(db.String(500),nullable = True)
+    # User details
+    name = db.Column(db.String(50), nullable=False)
 
-class User:
-    pass
-    # creating fields for user information
-    # email,#phone number(with county code) can be used as primary keys
-
-
-@app.route('/')
-def home():
-    return 'This is the home page'
-
-
-# creating decorator
-# can be used for email auth
-def token_existence(func):
+#creating decorator
+#can be used for email auth
+def token_existance(func):
```

Comment:
This function is great! Although decorators should be very clear and actionable. Can it be changed to `require_login`?